tests/cilium_http_integration_test.cc

```diff
@@ -1201,6 +1201,9 @@ TEST_P(SDSIntegrationTest, TestMissingSDSSecretOnUpdate) {
                  "of using gRPC",
                  temp_path, policy_path);
 
+  // Reduce flakiness by allowing some time for the policy to be updated before the following test
+  absl::SleepFor(absl::Milliseconds(100));
```

Comment:
is it easy to have this delay configured externally i.e. env, config, etc?

Reply:
Maybe, but hopefully that is not necessary, let's see how this goes first!